DeviceComunication/DeviceComunication.md

```diff
@@ -1119,18 +1049,16 @@ The following settings should be sufficient for normal use:
 | Medium    | 4        | 75       | 100      | 10       | 5        |
 | High      | 4        | 50       | 50       | 10       | 5        |
 
-##### Sampling period for temperature and humidity devices and how often to send a measure message
+##### Temperature and humidity devices's sampling period and setting the frequency of Measure messages.
```

Comment:
Odstavec níže - upraven v české verzi aby víc dával smysl. Mrknout na to a upravit podle toho ...